<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XPatchMakerClass</class>
 <widget class="QMainWindow" name="XPatchMakerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XPatchMaker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>811</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>平台</string>
    </property>
    <widget class="QComboBox" name="comboChannel">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>20</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>资源版本号:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAssetVer">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>文件数:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFileCount">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>提交服务器url:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUploadUrl">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>651</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>http://127.0.0.1/patch/us</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>本地路径:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLocalDir">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>511</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>C:\ff\</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>811</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>preupload</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>811</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>log</string>
    </property>
    <widget class="QListView" name="listViewLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>791</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>811</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>upload</string>
    </property>
    <widget class="QPushButton" name="pushButtonUpload">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>upload</string>
     </property>
    </widget>
    <widget class="QListView" name="listViewUploadEvent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>711</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonGenPatch">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>760</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>生成patch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCancel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>760</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMd5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>760</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>生成本地资源md5(仅首次使用)</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="xpatchmaker.qrc"/>
 </resources>
 <connections/>
</ui>
